src/index.js

```diff
@@ -6,12 +6,11 @@ class Experiment extends React.Component {
     super(props)
     this.state = {
       loading: true,
-      variant: null
+      variantName: null
     }
 
     this.experimentId = this.props.id
```

Comment:
This is not changed, but I don't think it's a good idea to store a `experimentId` on the component instance when we can already access it from `this.props`.

Reply:
And I believe this is probably why the component is not updating because the `experimentKey` is only updated at `constructor`.

Reply:
ok, I'll change it

Reply:
@fraserxu sorry, for some reason I missed your second message. No, the issue happens _not_ because the experimentKey is only set in constructor. Actually `experimentKey` should stay the same for one experiment, but I'd like not to do it this assignment in the constructor because _potentially_ this can cause the problem similar to what this PR fixes